Fix for recent Qwt 
See PR https://github.com/ElmerCSC/elmerfem/pull/294 ( Compatibility with Qwt 6.2. Fix for #293 #294 )
and commit https://github.com/ElmerCSC/elmerfem/commit/48e9430ccb858ca5bda28b967a0c84b51e2404b2
--- ElmerGUI/Application/src/convergenceview.h.orig	2020-11-10 20:52:44.000000000 +0100
+++ ElmerGUI/Application/src/convergenceview.h	2023-12-15 11:13:53.627242099 +0100
@@ -51,8 +51,8 @@
 #include <qwt_plot_curve.h>
 #include <qwt_plot_grid.h>
 #include <qwt_legend.h>
-/*#include <qwt_data.h> <-- deprecated in Qwt6, using qwt_compat.h instead*/
-#include <qwt_compat.h>
+/*#include <qwt_data.h> <-- deprecated in Qwt6, using qwt_compat.h instead
+#include <qwt_compat.h> <-- Removed in Qwt 6.2 */
 #include <qwt_text.h>
 #include <qwt_scale_engine.h>
 
@@ -76,8 +76,8 @@
   
 private:
   int d_count;
-  QwtArray<double> d_x;
-  QwtArray<double> d_y;
+  QVector<double> d_x;
+  QVector<double> d_y;
 };
 
 class Curve
